src/services/codefixes/importFixes.ts

```diff
@@ -864,9 +866,14 @@ function codeFixActionToCodeAction({ description, changes, commands }: CodeFixAc
 
 function getAllExportInfoForSymbol(importingFile: SourceFile | FutureSourceFile, symbol: Symbol, symbolName: string, moduleSymbol: Symbol, preferCapitalized: boolean, program: Program, host: LanguageServiceHost, preferences: UserPreferences, cancellationToken: CancellationToken | undefined): readonly SymbolExportInfo[] | undefined {
     const getChecker = createGetChecker(program, host);
+    const isFileExcluded = preferences.autoImportFileExcludePatterns && getIsFileExcluded(host, preferences);
+    const moduleSymbolExcluded = isFileExcluded && moduleSymbol.declarations && isSourceFile(moduleSymbol.declarations[0]) && isFileExcluded(moduleSymbol.declarations[0]);

```

Comment:
I think this needs to be something like “every declaration of `moduleSymbol` is excluded, by filename if SourceFile, or by containing SourceFile filename if a module declaration.”

Reply:
I was wrong about this; you can ignore my comment. If a module source file is excluded, augmentations to it in other files aren’t enough to make the module un-excluded.